src/models/Language.test.js

```diff
@@ -62,4 +62,13 @@ describe(`Language`, () => {
 
   });
 
+  describe(`Language.prototype.iso`, () => {
+    const lang = new Language();
+    it(`Language.prototype.iso`, function() {
+      expect(() => { lang.iso = `ctm`; }).not.toThrow();
+      expect(() => { lang.iso = `en`; }).toThrow();//e => e.name === `ISOCodeError`);
```

Comment:
You can / should check for the name of the error here, like we did in this other test:

https://github.com/digitallinguistics/javascript/blob/3bf13aad7fb4d201d9e203042d4b7db05a426285/src/models/Language.test.js#L60

Otherwise there might actually be some other error happening, and we wouldn't know that it's not the error we want to happen.